### PR TITLE
[RFC 54] Locale management UI

### PR DESCRIPTION
**Pull request by kaedroho**
Builds on #6220

### CODE REVIEW

File: wagtail/locales/templates/wagtaillocales/create.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{% extends "wagtailadmin/generic/create.html" %}
+
+{% block extra_js %}
+    {{ block.super }}
+    {% include "wagtailadmin/pages/_editor_js.html" %}
```
Comment (gasman) [inline wagtail/locales/templates/wagtaillocales/create.html:5]: Is this needed?

**Review by gasman — CHANGES_REQUESTED** [review]
Looks pretty good - just a minor quibble about the use of `_editor_js.html` (and imports need fixing to placate isort).

File: wagtail/locales/templates/wagtaillocales/create.html (hunk 1 above)
Comment (kaedroho) [reply wagtail/locales/templates/wagtaillocales/create.html]: nope, removed!

**Comment by gasman** [comment]
Looks like `wagtail.locales.utils.get_locale_usage` isn't playing well with [the one-off invalid model created in `TestSystemChecks`](https://github.com/wagtail/wagtail/blob/738003726900295328f263bfe91686026f70f596/wagtail/core/tests/test_translatablemixin.py#L142) - once that test has run, the model is returned in `get_translatable_models`, causing the subsequent query to fail as it doesn't have a table.

I'm thinking the best way around that is going to be either: find a way to un-register the invalid model after use (which looks a bit messy, involving fiddling around in internal properties of `AppConfig` and clearing the lru_cache on `apps.get_models`), or tweaking the test so that it temporarily modifies a real model from testapp.models rather than creating a new one.

**Comment by kaedroho** [comment]
@gasman thanks! That issue is fixed now

**Pull request merged**